code/modules/hydroponics/spreading/spreading.dm

```diff
@@ -314,4 +314,8 @@ var/list/global/cutoff_plant_icons = list()
 		else if (!reagents.get_free_space())
 			to_chat(usr, SPAN_NOTICE("It looks juicy."))
 		else
-			to_chat(usr, SPAN_NOTICE("It looks a bit dry."))
\ No newline at end of file
+			to_chat(usr, SPAN_NOTICE("It looks a bit dry."))
+
+/obj/effect/plant/is_drawable(allowmobs)
+	. = ..()
+	return TRUE
```

Comment:
this is not the correct way to do this. What you should do is modify the variable "reagent_flags".